packages/app-shell/src/common/utils/user.js

```diff
@@ -64,3 +66,7 @@ export function getUsersCurrentPlan(user) {
 
   return getSubscriptionPlanData(billingData);
 }
+
+export function hasPermissionToEdit(user) {
```

Comment:
This is not a blocking comment. I personally think that the name is not very representative of the privilege. 
```suggestion
export function hasPermissionToManageBilling(user) {
```

or something similar that is more descriptive could be more self-explained when an engineer sees it in use, without its implementation.

Reply:
@dinostheo I agree with you! good catch! I'll update this quickly then merge 😊